src/backend/cgobj.c

```diff
@@ -2312,10 +2312,11 @@ size_t Obj::mangle(Symbol *s,char *dest)
                 config.exe == EX_NT && tyfunc(s->ty()) &&
                 !variadic(s->Stype))
             {
+                size_t prelen = MARS ? 0 : 1;
                 dest[1] = '_';
-                memcpy(dest + 2,name,len);
-                dest[1 + 1 + len] = '@';
-                itoa(type_paramsize(s->Stype),dest + 3 + len,10);
```

Comment:
I have no idea what this set of changes accomplishes? It's also wrong, the ?: is backwards.